templates/firefox-ios.xliff

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?><xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 http://docs.oasis-open.org/xliff/v1.2/os/xliff-core-1.2-strict.xsd">
   <file original="Client/en.lproj/InfoPlist.strings" source-language="en" datatype="plaintext">
     <header>
-      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="12.4" build-num="12D4e"/>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="12.5" build-num="12E262"></tool>
     </header>
     <body>
       <trans-unit id="NSCameraUsageDescription" xml:space="preserve">
-        <source>This lets you take and upload photos.</source>
+        <source>Firefox uses your camera to scan QR codes and take photos and video.</source>
```

Comment:
String ID change needed.
Also, should it be "and videos" vs "video"? (since "photos" is plural here)

Reply:
Unfortunately we can't change the string key here as it is a system key. We're only changing it due to getting rejected during a release.

This string was provided by UX so I assume they don't want it plural. We can double check though

Reply:
I see. The thing is that if the string ID doesn't change, localizers don't get notified in Pontoon :/
Is that ok? This string may not be localized differently because of that